src/elements/common/Providers.tsx

```diff
@@ -10,7 +11,9 @@ const Providers = ({ children, hasProviders = true }: ProvideProps) => {
     if (hasProviders) {
```

Comment:
@greg-in-a-box I thought we added this file for customers. EUA shouldn't be using these providers because it already has the providers at the root of the parent app?

Reply:
thats correct, @kkuliczkowski-box this wont fix the issue in webapp, it only fixes the issue in BUIE via storybook, you will have to resolve this in webapp by using react portal logic without the use of the portal component here, as @tjuanitas mentioned we are moving away from BUIE components